### ISSUE TITLE
Add a second "job" similar to the test job that will "Milk the cows"

### ISSUE THREAD

**Issue by briggsrr** [ucsb-cs156-f22/f22-5pm-happycows#5]
The "jobs" controller has code that allows the user to start a "test job".

You are going to clone that code to add a job for "milking the cows".

So this code:
```
@ApiOperation(value = "Launch Test Job (click fail if you want to test exception handling)")
    @PreAuthorize("hasRole('ROLE_ADMIN')")
    @GetMapping("/launch/testjob")
    public Job launchTestJob(
        @ApiParam("fail") @RequestParam Boolean fail,
        @ApiParam("sleepMs") @RequestParam Integer sleepMs
    ) {
        return jobService.runAsJob(ctx -> {
            ctx.log("Hello World! from test job!");
            Thread.sleep(sleepMs);
            if (fail) {
                throw new Exception("Fail!");
            }
            ctx.log("Goodbye from test job!");
          });
    }
```

You'll copy/paste that, but:
- change the name from "Launch Test Job" to "Launch Job to Milk the Cows"
- remove the ApiParams (we don't need any params for this job)
- Inside, the code will just be as shown below:
```
return jobService.runAsJob(ctx -> {
            ctx.log("Starting to milk the cows");
            ctx.log("This is where the code to milk the cows will go.");
            ctx.log("Cows have been milked!");
          });
```

ALSO NOTE: the "code to milk the cows" is something that this team might implement, or that might be done by team 3; we'll provide more information about that in the next 2-3 days.

This is backend only. There will be a separate issue for the front end changes.

NOTE: once this issue is finished, do it again, but this time for "Update Cow Health".
Resist the temptation to do them both at the same time. We want to keep the pipeline flowing.
Do one, get it right, get it completely merged, then do the second one.
